### DISCUSSION TITLE
Mix old and new

### DISCUSSION THREAD

**Discussion by popey** [makealinuxapp/makealinuxapp#9] [General]
The old site was very much a single purpose site, with only one page.

In the new prototype which is currently published on gh pages, there is more of a documentation style.

I wonder if the news site should be a mix of the two?

A front-page which is a single page with attractive images, large fonts and only a small amount of text.

But all links dive deeper into a documentation style set of pages which elaborate on every topic, then sign post external resources?

What do we think?

**Comment by DanielDewberry** [comment]
I think your idea to mix the two styles is a good one. It's exactly what I hoped for.
